src/types/InventoryUnit.ts

```diff
@@ -34,3 +38,7 @@ export type InventoryUnitDataWrapper = {
 export type InventoryUnitData = {
   readonly data: InventoryUnitDataWrapper;
 };
+
+export type InventoryUnitActionRequest = {
+  readonly returnReason: string;
```

Comment:
what about exchange reason and cancel reason?

Reply:
![image](https://user-images.githubusercontent.com/59112643/183927253-7e9458e7-7211-4544-8fe6-156f644b623f.png)